docs/pages/build/internal-distribution.mdx

```diff
@@ -83,7 +83,7 @@ To share your app to Android devices, you must build an APK (Android application
 Apple offers [ad hoc provisioning profiles](https://help.apple.com/xcode/mac/current/#/dev7ccaf4d3c) to distribute your app to test devices once they have been registered
 to your Apple Developer account. This method requires a paid Apple Developer account and that account will only be able to use this method to distribute to at most 100 iPhones per year.
 
-You will need to know the UDID (Unique Device Identifier) of each device that will install your app, which may be challenging if you try to share with someone who is not a developer. Adding a new device will require a rebuild of your app.
+You will need to know the UDID (Unique Device Identifier) of each device that will install your app, which may be challenging if you try to share with someone who is not a developer. Adding a new device will require a rebuild of your app or [resigning the build](https://docs.expo.dev/app-signing/app-credentials/#re-signing-new-credentials).
```

Comment:
```suggestion
You will need to know the UDID (Unique Device Identifier) of each device that will install your app, which may be challenging if you try to share with someone who is not a developer. Adding a new device will require a rebuild of your app or [re-signing the build with new credentials](/app-signing/app-credentials/#re-signing-new-credentials).
```

- Let's expand the link to text for clarity, since this concept might be new for first-time developers
- Also, update the link to be relative since it's internal from the docs.